bower.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "webcomponents-platform",
+  "version": "1.0.0-rc.1",
+  "description": "Basic platform polyfills needed by webcomponents.js",
+  "main": "webcomponents-platform.js",
+  "authors": [
+    "The Polymer Project Authors (https://polymer.github.io/AUTHORS.txt)"
+  ],
+  "license": "BSD-3-Clause",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/webcomponents/webcomponents-platform.git"
+  },
+  "keywords": [
+    "web-components",
+    "webcomponents",
+    "polyfill",
+    "shim"
+  ],
+  "homepage": "https://webcomponents.org/polyfills",
+  "ignore": [
+    "**/.*",
```

Comment:
It's needed for excluding files like `.travis.yml`, `.gitignore`, etc. from Bower package.